shared/src/commonMain/kotlin/org/hyperskill/app/projects/domain/model/ProjectTracksEntry.kt

```diff
@@ -0,0 +1,12 @@
+package org.hyperskill.app.projects.domain.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ProjectTracksEntry(
+    @SerialName("level")
+    val level: ProjectLevel,
+    @SerialName("kind")
+    val kind: ProjectKind
```

Comment:
Maybe do `level` and `kind` nullable? For example, if the backend will add new types -> our HTPP request will fail with serialization error